### PR TITLE
Deploy Python tooling with pipenv

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -51,6 +52,21 @@ def run_full_build()
...
 
+def set_up_env_for_linux_pipenv(steps)
+{
+  withEnv([
+    "LC_ALL=C.UTF-8",
+    "LANG=C.UTF-8"])
+  {
+    steps()
+  }
+}
```
Comment (wkaluza) [inline Jenkinsfile]: I'm not sure if we should have this here or in the image. Here is less invasive, but then again not having a UTF-8 locale feels very last-century on the part of the image. Thoughts?

Comment (ejfitzgerald) [reply Jenkinsfile]: I have a mild preference for putting it in the image to be honest

**Review by ejfitzgerald — COMMENTED** [review]
Basically looks all good to me. As you suggested this will need to be circulated through the team but certainly sorts out the dependency issue quite nicely.

File: Jenkinsfile (hunk 1 above)
Comment (wkaluza) [reply Jenkinsfile]: I'll give it a go.

Comment (wkaluza) [reply Jenkinsfile]: Done.

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**